### ISSUE TITLE
Display notification when cluster is ready

### ISSUE THREAD

**Issue by jezdez** [mozilla/telemetry-analysis-service#213]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1324464

**Comment by marco-c** [comment]
It would be nice to use web-push here, there's a Python library at https://github.com/web-push-libs/pywebpush.

The README is short, but exhaustive. If you want to use it and you have questions, you can ask @jrconlin or me either here or on IRC.

**Comment by jezdez** [comment]
Yep, there is an easy to use Django app that uses pywebpush called [django-webpush](https://github.com/safwanrahman/django-webpush) that I looked at before. It's neat and I think it's just a matter of sitting down defining what notifications we want and then hooking it up. No need to reinvent the wheel. @safwanrahman has been a Mozilla contributor for a while and has been super helpful as well.

**Comment by rafrombrc** [comment]
Also see https://bugzilla.mozilla.org/show_bug.cgi?id=1316633.

**Comment by jezdez** [comment]
Since the last refactor of the cluster status system we can much easier now track the status, so tackling this would be much easier now.
